gel/_internal/_qbmodel/_abstract/_base.py

```diff
@@ -105,7 +105,15 @@ class GelObjectType(
     _qb.GelObjectTypeMetadata,
     metaclass=GelObjectTypeMeta,
 ):
-    pass
+    __gel_variant__: ClassVar[str | None] = None
```

Comment:
I don't understand the need of `__gel_variant__`. Is it just a marker on a type?

Reply:
Though actually, it would be quite annoying to have it as an enum, because variants are inherently target-specific (e.g Pydantic) and the field is defined in the "abstract" query-builder model.  I'd leave it as a string for now.